import-service/serverless.ts

```diff
@@ -93,9 +119,16 @@ const serverlessConfiguration: Serverless = {
       events: [
         {
           http: {
-            method: "get",
+            method: "put",
             path: "import",
             cors: true,
+            authorizer: {
+              name: "basicAuthorizer",
+              arn:
+                "arn:aws:lambda:us-east-1:443507229806:function:authorization-service-dev-basicAuthorizer",
```

Comment:
To avoid hardcoded value here you can use export/import
in authorization service you can write something like:
```
resources: {
    Resources: {},
    Outputs: {
      AuthorizationARN: {
        Value: {
          'Fn::GetAtt': ['BasicAuthorizerLambdaFunction', 'Arn']
        },
        Export: {
          Name: 'AuthorizationARN',
        }
      }
    }
  },
```
and then in import service for 'custom' field

```
authorizerArn: {
   'Fn::ImportValue': 'AuthorizationARN',
},
```

and here:

`arn: '${self:custom.authorizerArn}',`